golang/inbound.go

```diff
@@ -91,7 +90,12 @@ func (c *Connection) handleCallReq(frame *Frame) bool {
 	response.messageForFragment = func(initial bool) message {
 		if initial {
 			call.AddAnnotation(AnnotationKeyServerSend)
-			call.Report(callReq.Tracing, c.traceReporter)
+			targetEndpoint := TargetEndpoint{
```

Comment:
to be consistent, let's either create `targetEndpoint` either inside `messageForFragment` or outside of it in both `inbound.go` and `outbound.go`